### PR TITLE
Wurst tracking beacon

### PR DESCRIPTION
**Pull request by theQuazz**

### CODE REVIEW

File: wurst/lib/PingMinimap.wurst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package PingMinimap
+
+let DEFAULT_DURATION = 10.0
+
+public function pingMinimap(vec2 pos, real duration, color color, bool extraEffects)
```
Comment (jonathanmclaus) [inline wurst/lib/PingMinimap.wurst]: Is `extraEffects` only for the exclamation? I think an enum for the final parameter might make it more readable.

File: wurst/lib/PingMinimap.wurst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    PingMinimapEx(pos.x, pos.y, duration, color.red, color.green, color.blue, extraEffects)
+
+public function pingMinimap(vec2 pos, real duration, color color)
+    pingMinimap(pos, duration, color, false)
+
+public function pingMinimap(vec2 pos, color color)
+    pingMinimap(pos, DEFAULT_DURATION, color)
+
+public function pingMinimap(vec2 pos, real duration)
```
Comment (jonathanmclaus) [inline wurst/lib/PingMinimap.wurst]: How does this work without a color?

File: wurst/systems/spells/TrackerBeacon.wurst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    if beacons.has(owner)
+        beacons.remove(owner)
+
+    let beacon = new Beacon(target.getOwner())
+    beacons.put(owner, beacon)
+
+function queryBeacon()
+    let owner = GetSpellAbilityUnit().getOwner()
+
+    if not beacons.has(owner)
```
Comment (jonathanmclaus) [inline wurst/systems/spells/TrackerBeacon.wurst]: Can you use a lookup without checking this and then compare the result to `null`?

**Comment by theQuazz** [comment]
@jonathanmclaus updated

**Pull request merged**